engine/tests/ElggCoreGetEntitiesTest.php

```diff
@@ -348,6 +348,40 @@ public function testElggAPIGettersTSPValidTypeValidSubtype() {
 	}
 
 	/**
+	 * Valid type, ELGG_ENTITIES_NO_VALUE as subtype
+	 */
+	public function testElggAPIGettersTSPValidTypeNoValueSubtype() {
```

Comment:
The existing functions also have confusing names. I suppose this is fine for now. Let's make all the names better in a separate PR.